src/assets/TypeScriptAsset.js

```diff
@@ -28,9 +31,41 @@ class TypeScriptAsset extends Asset {
 
     // Overwrite default if config is found
     if (tsconfig) {
+      // Files read by TypeScript parser while resolving "extends"
+      let readFiles = [];
+      // https://github.com/Microsoft/TypeScript/blob/62306bc3f9e119dc17b400ff263dad532da6cdb1/scripts/build/upToDate.js#L370
+      const parseConfigHost = {
+        useCaseSensitiveFileNames: true,
+        getCurrentDirectory: () => process.cwd(),
```

Comment:
Is this right? Should it be relative to the current file being transformed?

Reply:
I looked into it, and it seems that some properties are not required including this. https://github.com/Microsoft/TypeScript/blob/29dd67ec40361db26612a86aa6bb52ca5e7f34a8/src/compiler/types.ts#L2713
I'll remove this and see if it works fine.

Reply:
@devongovett Ended up using TypeScript API that generates host.